Sources_v3/Query/ChannelMemberListQuery.swift

```diff
@@ -4,8 +4,15 @@
 
 import Foundation
 
+/// A namespace for the `FilterKey`s suitable to be used for `ChannelMemberListQuery`. This scope is not aware of any
+/// extra data types.
+public typealias AnyMemberListFilterScope = AnyUserListFilterScope
+
+/// An extra-data-specific namespace for the `FilterKey`s suitable to be used for `ChannelMemberListQuery`.
+public typealias MemberListFilterScope<ExtraData: UserExtraData> = UserListFilterScope<ExtraData>
```

Comment:
`MemberListFilterScope` should have `is_moderator` and `invite` filters too: https://getstream.io/chat/docs/query_members/?language=swift

Reply:
Well, that's a complication 🤔

Reply:
Heh, well, it wasn't a complication at all. This system is more robust and flexible than I thought 🎉 Please check 
156e2e4 , I didn't create fixups because it's easier to review like this.